hip-more-cultural-names/common/landed_titles/0_HIP_MoreCulturalNames.txt

```diff
@@ -12470,15 +14336,47 @@ e_byzantium = {
                     thuringian = "Welbaschd" # Historical? Copied from German
                 }
                 b_zemen = {
+                    bavarian = "Belowo" # Historical? Copied from German
                     bosnian = "Bilovo"
+                    carantanian = "Bilovo" # Historical? Copied from Croatian
```

Comment:
all the other south slavs should have "Belovo" idk why you deleted them

Reply:
"Belovo" is the default name for all languages